lib/routes/account.js

```diff
@@ -410,6 +410,8 @@ module.exports = function (
           })
         }
 
+        request.emitMetricsEvent('flow.signin.submit')
```

Comment:
Maybe we should just delete this event, for now at least? We have an `attempt` event being emitted by the content server immediately prior to the request being sent, which combined with any error events we add to this route in due course should cover most of the drop-offs.